Dockerfile

```diff
@@ -1,20 +1,41 @@
-FROM alpine:3.6
+
+###################################
+#Build stage
+FROM golang:alpine AS build-env
```

Comment:
It is a bad practice not to fix on specific version number. Besides, `golang:alpine` is the same as `golang:1.9-alpine-3.6`. You can change it to `alpine:3.7` and install `go` package with `apk` from Alpine's community repository.

Reply:
I know, I hesitate on that part and thinking that we it's not that bad since it is only for build. 
You are rigth, I will fix to 3.7 and use in go alpine packages (1.9) and when golang:1.10-alpine-3.7 is release I will update. It's too bad that official image doesn't offer a golang:1.9-alpine-3.7 tag.

Reply:
A golang:1.9-alpine-3.7 tag should be release soon. ^^